src/openrct2/network/Http.h

```diff
@@ -0,0 +1,83 @@
+#pragma region Copyright(c) 2014 - 2018 OpenRCT2 Developers
+/*****************************************************************************
+ * OpenRCT2, an open source clone of Roller Coaster Tycoon 2.
+ *
+ * OpenRCT2 is the work of many authors, a full list can be found in contributors.md
+ * For more information, visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * A full copy of the GNU General Public License can be found in licence.txt
+ *****************************************************************************/
+#pragma endregion
+
+#ifndef OPENRCT2_NETWORK_HTTP_H_
+#define OPENRCT2_NETWORK_HTTP_H_
+
+#ifndef DISABLE_HTTP
+
+#include <atomic>
```

Comment:
Doesn't look like this header is needed here?

Reply:
Right it's a relict...